mindmeld/core.py

```diff
@@ -229,6 +240,41 @@ def __init__(
         self._timestamp = timestamp
         self.stemmed_tokens = stemmed_tokens or tuple()
 
+    def char_maps_to_cache(self):
+        return {json.dumps(k) : v for k,v in self._char_maps.items()}
```

Comment:
Since the keys are tuples, why cant we do this (since json.dump has some unsavory mem footprint):

Serialization: `return {str(k) : v for k,v in self._char_maps.items()}`

Deserialization of the tuple:`tuple(map(int, str.split(', ')))`